### DISCUSSION TITLE
Customise the bottom search bar

### DISCUSSION THREAD

**Discussion by William150722** [LawnchairLauncher/lawnchair#2942] [Q&A] [answered]
### About this discussion

![screenshot-Samsung Galaxy S22 Plus-12 0](https://user-images.githubusercontent.com/111726325/190052856-06ff9e47-7d0a-4479-aba6-83fe7053af8c.png)

Do anyone have idea how to customise the bottom search bar here?

**Comment by SuperDragonXD** [comment]
Go to Home Settings > Dock. You can customize the search provider, theme the search bar, and change the corner radius. You can also hide the search bar there.

**Reply by William150722** [reply]
@SuperDragonXD 
I means in remove the google mic icon or change the google icon add some text label as well.

**Reply by SuperDragonXD** [reply, marked as answer]
Oh that. For now, we currently don't have the ability to customize the individual components of the search bar.

**Reply by kahanzammi** [reply]
I guess the best solution is to allow to add a custom 4x1 widget to the dock (to replace current search bar).
There are  widgets in play store (like https://play.google.com/store/apps/details?id=rk.android.app.searchbarwidget&hl=en&gl=US) ideal replacement candidates for this.
